opensrp-eusm/src/main/java/org/smartregister/eusm/interactor/LoginInteractor.java

```diff
@@ -27,9 +26,6 @@ protected void scheduleJobsPeriodically() {
         ImageUploadServiceJob.scheduleJob(ImageUploadServiceJob.TAG,
                 BuildConfig.PULL_UNIQUE_IDS_MINUTES, getFlexValue((int) BuildConfig.PULL_UNIQUE_IDS_MINUTES));
 
-        SyncStockServiceJob.scheduleJob(SyncStockServiceJob.TAG,
-                BuildConfig.PULL_UNIQUE_IDS_MINUTES, getFlexValue((int) BuildConfig.PULL_UNIQUE_IDS_MINUTES));
-
```

Comment:
Why is this being removed?

Reply:
Moved it to `AppLocationIntentService` so it will be called when the `LocationService` Job runs.
https://github.com/OpenSRP/opensrp-client-eusm/blob/additional-work-stock-locations/opensrp-eusm/src/main/java/org/smartregister/eusm/service/AppLocationTaskIntentService.java